docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -70,7 +75,9 @@ curl -sSLf https://get.openziti.io/tun/scripts/install-ubuntu.bash | bash
 | 10 Buster   | Bionic 18.04 | x86_64        |
 |  9 Stretch  | Xenial 16.04 | x86_64        |
 
-This example subscribes you to the Ubuntu `focal` repo which will work well in most cases. Alternatively, you may refer to the table to find the Ubuntu release name that is the contemporary of your Debian release. Then, substitute the Ubuntu release name for `focal` in the `/etc/apt/sources.list.d/openziti.list` file.
+This example subscribes you to the Ubuntu `focal` repo which will work well in most cases. Alternatively, you may refer
```

Comment:
Often times I'll scan my own doc (i mean, i forget to do it sometimes) for "you" and I often replace it with "to do ${x}, run ${y}" type  of wording:

That means you can avoid words that are extra such as:
```
This example subscribes you to the Ubuntu `focal` repo which will work well in most cases.
```
I tend to avoid making any kinds of recommendations like this too. just state "To install on `focal`, run `blah`" type of words